apps/spruce/src/pages/spawn/spawnHost/SpawnHostActionButton.tsx

```diff
@@ -106,10 +106,12 @@ export const SpawnHostActionButton: React.FC<{ host: MyHost }> = ({ host }) => {
 
   return (
     <>
-      {/* TODO: Replace with noExpiration check when sleep schedules are deployed */}
-      {/* @ts-expect-error: FIXME. This comment was added by an automated script. */}
-      {!isNullSleepSchedule(host?.sleepSchedule) &&
-      action === SpawnHostStatusActions.Stop ? (
+      {isSleepScheduleActive({
+        noExpiration: host.noExpiration,
+        permanentlyExempt: !!host?.sleepSchedule?.permanentlyExempt,
+        temporarilyExemptUntil: host?.sleepSchedule
+          ?.temporarilyExemptUntil as unknown as string,
```

Comment:
would it be bad to just turn this parameter into a boolean? Could maybe change the variable to be called temporilyExempt or something like that
```suggestion
        temporarilyExempt: !!host?.sleepSchedule?.temporarilyExemptUntil,
```